src/NLog.Web/README.md

```diff
@@ -0,0 +1,30 @@
+# NLog.Web
+
+- Logger Provider for ASP.NET platform.
+- Log all properties of HttpContext and HostingEnvironment via [layout renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web) just by changing the NLog.config file.
+- Capable to log to many destinations via [targets](https://nlog-project.org/config/?tab=targets).
+- Supports common [layouts](https://nlog-project.org/config/?tab=layouts), such as CVS, JSON, W3C ELF, and XML.
```

Comment:
"Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog Layouts"

Reply:
Completed